src/App.tsx

```diff
@@ -15,33 +17,170 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <div className="App">
-    <button type="button">
-      Start
-    </button>
-
-    <button type="button">
-      Sort alphabetically
-    </button>
-
-    <button type="button">
-      Sort by length
-    </button>
-
-    <button type="button">
-      Reverse
-    </button>
-
-    <button type="button">
-      Reset
-    </button>
-
-    <ul className="Goods">
-      <li className="Goods__item">Dumplings</li>
-      <li className="Goods__item">Carrot</li>
-      <li className="Goods__item">Eggs</li>
-      <li className="Goods__item">...</li>
-    </ul>
-  </div>
-);
+enum SortType {
+  NONE,
+  ALPABET,
+  LENGTH,
+}
+
+type State = {
+  isStarted: boolean,
+  isReversed: boolean,
+  isSortAlphabet: boolean,
+  isSortLength: boolean,
+  sortType: SortType,
+  valueMinLensthGood: number,
+};
+
+export const App: React.FC<State> = () => {
+  const [isStarted, setIsStarted] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
+  const [isReversed, setIsReversed] = useState(false);
+  const [valueMinLensthGood, setValue] = useState(1);
+  const [isSortAlphabet, setIsSortAlphabet] = useState(false);
+  const [isSortLength, setIsSortLength] = useState(false);
```

Comment:
Why do you need these states? You already have [sortType] where you are saving needed info.
```suggestion
```